charts/prl-cos/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "1.0"
 description: A Helm chart for prl-cos-api App
 name: prl-cos
 home: https://github.com/hmcts/prl-cos-api
-version: 0.0.56
```

Comment:
any idea why chart version upgrade is needed?

Reply:
Synced with master, so reverted.